adapters/rubicon/rubicon.go

```diff
@@ -247,6 +248,12 @@ type rubiconUserExtEidUidExt struct {
 	RtiPartner string `json:"rtiPartner,omitempty"`
 }
 
+type mappedRubiconUidsParam struct {
+	tpIds       []rubiconExtUserTpID
```

Comment:
Does tp = third party? If so, please consider naming the `thirdPartyIDs` for clarity.

Reply:
@SyntaxNode 
This name is already self-completed. It is used in Rubicon Bidder and (for my knowledge) does not mean thirdParty. 
Thanks for proposal

Reply:
Alright.